docs/conf.py

```diff
@@ -48,7 +49,9 @@
 
 # General information about the project.
 project = u'Pulp Project'
-copyright = u'2012-2014, Pulp Team'
+
+# Set copyright to current year
+copyright = u'2012-' + str(date.today().year).decode("utf-8") + u', Pulp Team'
```

Comment:
Since this isn't Python 3 compatible anyway, it might be better to be explicit and cast ``date.today().year`` to unicode directly ``unicode(date.today().year)``